docs/Configuring.md

```diff
@@ -387,3 +388,23 @@ server:
 #### Managing Authorization Policy
 
 Admins can manage the authorization policy directly in the YAML configuration file. For detailed configuration options, refer to the [Casbin documentation](https://casbin.org/docs/en/syntax-for-models).
+
+## Cache Configuration
+
+The platform supports a cache manager to improve performance for frequently accessed data. You can configure the cache backend and its resource usage.
+
+Root level key `cache`
+
+| Field                    | Description                                                      | Default      |
+|--------------------------|------------------------------------------------------------------|--------------|
+| `driver`                 | Type of cache backend (`ristretto`)                              | `ristretto`  |
+| `ristretto.maxCost`      | Maximum cost for the cache (e.g. 100mb, 1gb)                     | `8gb`        |
+
+Example:
+
+```yaml
+cache:
+  driver: ristretto           # Type of cache backend (currently only 'ristretto' is supported)
+  ristretto:
+    maxCost: 1gb              # Maximum cost (in bytes) for the cache (default: 8GB)
+```
```

Comment:
can we add docs for how to configure expiration and cost for a specific service via the service config yaml?